hera/editor/behavior/DesignBehavior.tsx

```diff
@@ -228,11 +234,47 @@ export default class DesignBehavior {
     return null;
   }
 
+  private putMultiple(
+    vectors: Vector[],
+    state: State,
+    actions: Actions,
+    editor: EditorState,
+  ): StateLike | null {
+    let result: StateLike | null = null;
+    const players = Array.from(
+      new Set([...state.map.active, ...PlayerIDs]),
+    ).slice(0, vectors.length);
+    vectors.forEach((vector, index) => {
+      const currentPlayerIndex = players.indexOf(
+        state.map.getCurrentPlayer().id,
+      );
+      const playerId =
+        players[
+          ((currentPlayerIndex >= 0 ? currentPlayerIndex : 0) + index) %
+            players.length
+        ];
```

Comment:
This should not include `0` as `0` is the neutral unit type that can be rescued. Can you update it so it only uses one of the valid player ids except 0?

Reply:
ec683d3fb1f0778594b4044b7cf4060ac628d859